src/sas/sascalc/dataloader/readers/IgorReader.py

```diff
@@ -113,14 +113,14 @@ def read(self, filename=None):
                 # Get total bin number
                 
             i_tot_row += 1
-        i_tot_row = math.ceil(math.sqrt(i_tot_row)) - 1
-        #print "i_tot", i_tot_row
+        i_tot_row = int(math.ceil(math.sqrt(i_tot_row)) - 1)
+        # print "i_tot", i_tot_row

```

Comment:
Agree with removing the print statement.
I don't think that rounding to the nearest integer is what is needed here. The nearest integer closer to zero is required. It could also be written

    i_tot_row = int(math.floor(math.sqrt(i_tot_row)))

Perhaps that's a better way of doing it because if the detector is a perfect square (e.g. 16384) then `int(math.ceil(math.sqrt(i_tot_row)) - 1)` is 127 and `int(math.floor(math.sqrt(i_tot_row)))` is 128.